client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx

```diff
@@ -48,150 +48,116 @@ export default function MigrationsFAQs() {
+			),
+		},
+		{
+			id: 'buyout-credit-calculation',
+			question: translate( 'How do you calculate my buyout credit?' ),
+			answer: translate(
+				'We will ask you to provide a copy of your current contract or invoice to determine the remaining time on your plan. We will then credit you for the same amount of time left on the plan that you choose in Automattic for Agencies.'
+			),
+		},
+		{
+			id: 'multiple-sites-migration-offers',
+			question: translate( 'Are there any special offers for agencies moving multiple sites?' ),
+			answer: (
+				<>
+					{ translate(
+						'From now until the end of 2024, you’ll receive $100 for each site you migrate to Pressable or WordPress.com, up to $10,000*. If you’re a WP Engine customer, we’ll also credit the costs to set you free.'
+					) }
+					<br />
+					<br />
+					{ translate(
+						'* The migration limit is $10,000 for WP Engine and $3,000 for other hosts.'
+					) }
+				</>
+			),
+		},
+		{
+			id: 'migration-tasks-to-handle-on-my-own',
+			question: translate( 'What tasks will I need to handle on my own during the migration?' ),
+			answer: translate(
+				'If you choose to have our migration team handle the actual migrations, you will need to initially set up a login on each site and share it with them (we provide step-by-step instructions and a secure method to share). Once the migration is completed, our team reviews the copy of the site on our servers and will ask you to do the same. You’ll then need to point the domain to our servers by changing the DNS records.'
+			),
+		},
+		{
+			id: 'migration-host-application',
+			question: translate( 'Does this apply to any host?' ),
+			answer: translate(
+				'Yes. It doesn’t matter where your site is hosted; our migration offer extends to any site migrated to WordPress.com or Pressable.'
+			),
+		},
+		{
+			id: 'migration-existing-host-location',
+			question: translate( 'Does my site already have to be hosted on WordPress?' ),
+			answer: translate(
+				'No. It doesn’t matter what CMS your site currently uses. You will be eligible for our migration offer if you migrate your site to WordPress.com or Pressable. However, we won’t be able to provide you with migration assistance in this case.'
+			),
+		},
+		{
+			id: 'migration-payment-process',
+			question: translate( 'How do I get paid?' ),
+			answer: translate(
+				'Once your sites have been active on WordPress.com or Pressable hosting for over 6 months, we will pay you the site migration commission during the payout windows listed {{PayoutKBLink}}here{{/PayoutKBLink}}. You will also need to add your payment details to our payment partner, Tipalti, which you can do {{paymentSettingLink}}here{{/paymentSettingLink}}.',
```

Comment:
[NITPICK]

Sorry for not noticing this earlier. Feel free to ignore this.

I think it is better to keep the component name consistent here.

```suggestion
				'Once your sites have been active on WordPress.com or Pressable hosting for over 6 months, we will pay you the site migration commission during the payout windows listed {{PayoutKBLink}}here{{/PayoutKBLink}}. You will also need to add your payment details to our payment partner, Tipalti, which you can do {{PaymentSettingLink}}here{{/PaymentSettingLink}}.',
```

Reply:
Good catch, yes. This should be Capitalize 👍